docs/developers/documentation/index.md

```diff
@@ -19,7 +19,7 @@ If you'd like to contribute a brand new document to our usage section, it might
 The napari team can also suggest what type of document would be best suited, and whether there are already
 existing documents that could be expanded to include the content you think is lacking.
 
-Examples of documents you might want to contribute are:
+Go to your local napari folder to find examples of documents you might want to contribute. The paths are listed in parentheses below.
```

Comment:
This description is still nonclear to me. In my opinion, it assumes that the person install napari from the repository, not `pip` or `conda`

Reply:
@Czaki since this is part of the contributing documentation guide, I think that's a valid assumption in this case.